File: fortran/fizzbuzz.f90
program main
      implicit none
      integer i
      logical m3, m5
      do i=1,100
        m3 = mod(i, 3) == 0
        m5 = mod(i, 5) == 0
        if (m3 .and. m5) then
          print "(a8)", "FizzBuzz"
        else if (m3) then
          print "(a4)", "Fizz"
        else if (m5) then
          print "(a4)", "Buzz"
        else if (i<10) then
          print '(i1)', i
        else
          print '(i2)', i
        end if
      end do
end program

File: python/fizzbuzz.py
msg = {3: "Fizz", 5: "Buzz"}

for i in range(1,101):
    activated = False
    for j in m:
        if not i % j:
            activated = True
            print(msg[j], end="")

    end = "" if activated else i
    print(end)
